packages/hardhat/test/Challenge3.ts

```diff
@@ -20,9 +20,10 @@ describe("🚩 Challenge 3: 🎲 Dice Game", function () {
     const DiceGame = await ethers.getContractFactory("DiceGame");
     diceGame = await DiceGame.deploy();
 
+    const contractAddress = process.env.CONTRACT_ADDRESS;
     let contractArtifact;
     if (process.env.CONTRACT_ADDRESS) {
```

Comment:
To be consistent with other challenges, maybe we can change this to `contractAddress`

Reply:
Ohh yes my bad forge to update it, fixed at https://github.com/scaffold-eth/se-2-challenges/pull/141/commits/c603e9a9816a59a5d9be60ed32789e66721c32cd 🙌